spec/spec_helper.rb

```diff
@@ -3,7 +3,7 @@
 require 'openstates'
 require 'pry'
 require 'pry-remote'
-require 'pry-debugger'
+require 'pry-byebug'
```

Comment:
This will also need changed in the `spec_helper.rb` as it explicitly includes the `pry-debugger` gem on line 6.